### PR TITLE
Add Employee Managment system

### PR DESCRIPTION
**Pull request by bulgarchik**

### CODE REVIEW

File: FastBank.Infrastructure/DTOs/EmployeeDTO.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using FastBank.Domain;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace FastBank.Infrastructure.DTOs
+{
+    [Table("Emoyees")]
```
Comment (AngelovGrowth) [inline FastBank.Infrastructure/DTOs/EmployeeDTO.cs:7]: Table name "Emoyees"?

File: FastBank.Infrastructure/Repository/EmployeeRepository.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+using FastBank.Infrastructure.Context;
+using FastBank.Infrastructure.DTOs;
+
+namespace FastBank.Infrastructure.Repository
+{
+    public class EmployeeRepository : IEmployeeRepository
+    {
+        private readonly IRepository _repo;
+
+        public const int EmployeesPerPage = 3;
```
Comment (AngelovGrowth) [inline FastBank.Infrastructure/Repository/EmployeeRepository.cs:12]: Constants are written differently. First, constant if it is strongly specific for the class, should be private not public.
Second, constants are commonly written like this: EMPLOYEES_PER_PAGE = 3

File: FastBank.Infrastructure/Repository/EmployeeRepository.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+        public EmployeeRepository()
+        {
+            _repo = new Repository(new FastBankDbContext());
+        }
+        public void AddEmployee(Employee employee)
```
Comment (AngelovGrowth) [inline FastBank.Infrastructure/Repository/EmployeeRepository.cs:18]: Add space please

File: FastBank.Infrastructure/Repository/EmployeeRepository.cs (hunk 2 above)
Comment (bulgarchik) [reply FastBank.Infrastructure/Repository/EmployeeRepository.cs:12]: image
Can't change this constant to private use it in another method

File: FastBank.Infrastructure/Repository/EmployeeRepository.cs (hunk 3 above)
Comment (bulgarchik) [reply FastBank.Infrastructure/Repository/EmployeeRepository.cs:19]: fixed

**Pull request merged**